src/EventListener/TwoFactorLoginSuccessHandler.php

```diff
@@ -45,14 +46,21 @@
      */
     private $userManager;
 
+    /**
+     * @var RouterInterface
```

Comment:
You should use `UrlGeneratorInterface` and rename the variables accordingly

Reply:
sure

Reply:
Fixed